.idea/.gitignore

```diff
@@ -0,0 +1,2 @@
+# Default ignored files
```

Comment:
I prefer all the `.idea/` dir will get inside the `.gitignore`, I don't like any IDE-specific files inside repo.

Reply:
At least new line at end of file

Reply:
Added it